design/index.js

```diff
@@ -6,6 +6,7 @@ export Card from './card'
 export CloseIcon from './close-icon'
 export Drawer from './drawer'
 export Logo from './logo'
+export Numeral from './numeral'
```

Comment:
@Joozty do you have any better ides if Number is reserved?

Reply:
I don't think there is anything better for this. Maybe `Digit`?